scale-tests/configs/2018-08-08-mvrandom110s-full-scale.env

```diff
@@ -0,0 +1,111 @@
+INFRASTRUCTURE_OUTPUT_FILE="${TEST_NAME}-infrastructure.json"
+KAFKA_ZOOKEEPER_CONFIG='scale-tests/configs/kafka-zookeeper-options.json'
+KAFKA_CLUSTER_COUNT=1
+KAFKA_CONFIG='scale-tests/configs/kafka-options.json'
+CASSANDRA_CLUSTER_COUNT=1
+CASSANDRA_CONFIG='scale-tests/configs/cassandra-options.json'
+
+# Non-GPU dispatchers configuration ############################################
+
+NON_GPU_NUM_DISPATCHERS=50
+NON_GPU_DISPATCHERS_OUTPUT_FILE="${TEST_NAME}-non-gpu-dispatchers-${NON_GPU_NUM_DISPATCHERS}.out"
+NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE="${NON_GPU_DISPATCHERS_OUTPUT_FILE}-dispatchers.json" # NOTE: this name is set internally by the deploy dispatchers script
+NON_GPU_QUOTA_DRIVERS_CPUS=20
+NON_GPU_QUOTA_DRIVERS_MEM=50000
+NON_GPU_QUOTA_EXECUTORS_CPUS=25
+NON_GPU_QUOTA_EXECUTORS_MEM=40000
+
+# GPU dispatchers configuration ################################################
+
+GPU_NUM_DISPATCHERS=5
+GPU_DISPATCHERS_OUTPUT_FILE="${TEST_NAME}-gpu-dispatchers-${GPU_NUM_DISPATCHERS}.out"
+GPU_DISPATCHERS_JSON_OUTPUT_FILE="${GPU_DISPATCHERS_OUTPUT_FILE}-dispatchers.json" # NOTE: this name is set internally by the deploy dispatchers script
+GPU_QUOTA_DRIVERS_CPUS=16
+GPU_QUOTA_DRIVERS_MEM=200000
+GPU_QUOTA_EXECUTORS_CPUS=
+GPU_QUOTA_EXECUTORS_MEM=
+GPU_QUOTA_EXECUTORS_GPUS=
+# NOTE: to test Core team’s hypothesis of GPU quota negatively impacting spark
+# launch rates, we remove the GPU quota entirely from all executor roles.
+GPU_REMOVE_EXECUTORS_ROLES_QUOTAS=true
+
+# Common streaming jobs configuration ##########################################
+
+TEST_ASSEMBLY_JAR_URL='http://infinity-artifacts.s3.amazonaws.com/scale-tests/dcos-spark-scala-tests-assembly-20180612-5fa9420.jar'
+NUM_DISPATCHERS="$((${NON_GPU_NUM_DISPATCHERS} + ${GPU_NUM_DISPATCHERS}))"
+DISPATCHERS_JSON_OUTPUT_FILE="${TEST_NAME}-all-dispatchers-${NUM_DISPATCHERS}.json"
+
+# Failing streaming jobs configuration #########################################
+
+FAILING_SUBMISSIONS_OUTPUT_FILE="${TEST_NAME}-failing-submissions.out"
+FAILING_NUM_PRODUCERS_PER_KAFKA="${NUM_DISPATCHERS}" # 1 Kafka and 50 dispatchers -> 50 producers.
+FAILING_NUM_CONSUMERS_PER_PRODUCER=1 # 50 producers -> 50 consumers.
+FAILING_PRODUCER_NUMBER_OF_WORDS=1300000
+FAILING_PRODUCER_WORDS_PER_SECOND=1
+FAILING_PRODUCER_SPARK_CORES_MAX=2
+FAILING_PRODUCER_SPARK_EXECUTOR_CORES=2
+FAILING_CONSUMER_BATCH_SIZE_SECONDS=10
+FAILING_CONSUMER_SPARK_CORES_MAX=1
+FAILING_CONSUMER_SPARK_EXECUTOR_CORES=1
+
+# Finite streaming jobs configuration ##########################################
+
+FINITE_SUBMISSIONS_OUTPUT_FILE="${TEST_NAME}-finite-submissions.out"
+FINITE_NUM_PRODUCERS_PER_KAFKA="${NUM_DISPATCHERS}" # 1 Kafka and 50 dispatchers -> 50 producers.
+FINITE_NUM_CONSUMERS_PER_PRODUCER=1 # 50 producers -> 50 consumers.
+# 50 producers + 50 consumers = 100 total finite streaming jobs
+FINITE_PRODUCER_NUMBER_OF_WORDS=1300000
+FINITE_PRODUCER_WORDS_PER_SECOND=1
+# 1300000 words / 1 word per second -> ~2h runtime.
```

Comment:
Comment to myself, this should be `7692` for a similar runtime to the 13 words per second case.